lib/resources/oidc.js

```diff
@@ -105,17 +108,19 @@ module.exports = (service, endpoint) => {
 
       const code_challenge = generators.codeChallenge(code_verifier); // eslint-disable-line camelcase
 
+      const { next } = req.query ?? '';
```

Comment:
This may not do what it intends to... I suspect this is different to:

```js
const next = req.query.next ?? '';
```

Maybe add a test for `next` missing, if that's a possible situation.